configs/sites/discover/packages.yaml

```diff
@@ -43,6 +37,44 @@ packages:
       prefix: /discover/swdev/jcsda/spack-stack/miniconda-3.9.7/
       modules:
       - miniconda/3.9.7
+###   glib:
```

Comment:
Note. This junk will be removed later if not in a last minute doc push to this PR.